SPECS/python-platformdirs/python-platformdirs.spec

```diff
@@ -1,5 +1,4 @@
 # Disable tests as it requires new package python-exceptiongroup
-%global with_check 0
 %global srcname platformdirs
 %bcond_without tests
```

Comment:
Please remove as well - that's not how running tests is controlled in our build infrastructure and this will just override the global settings.